docs/core/windows-prerequisites.md

```diff
@@ -4,54 +4,89 @@ description: Learn what dependencies you need on your Windows machine to develop
 keywords: .NET Core, Windows, prerequisites, dependencies, Visual Studio
 author: mairaw
 ms.author: mairaw
-ms.date: 06/26/2017
+ms.date: 08/09/2017
 ms.topic: article
 ms.prod: .net-core
 ms.devlang: dotnet
 ms.assetid: c33b1241-ab66-4583-9eba-52cf51146f5a
 ---
 
-# Prerequisites for .NET Core on Windows
+# Prerequisites for .NET Core on windows
 
-This article shows you what dependencies you need to deploy and run .NET Core applications on Windows machines and develop using Visual Studio.
+This article shows the needed dependencies to develop .NET Core applications on Windows. The supported OS versions and dependencies that follow apply to the three ways of developing .NET Core apps on Windows:
 
-## Supported Windows versions
+* [Command line](tutorials/using-with-xplat-cli.md)
+* [Visual Studio 2017](https://www.visualstudio.com/downloads/)
+* [Visual Studio Code](https://code.visualstudio.com/)
 
-.NET Core is supported on the following versions of Windows:
+##.NET Core 2.x/1.x supported windows versions
```

Comment:
add space between # and heading titles

.NET Core 2.x/1.x supported windows versions -> .NET Core supported Windows versions